tron/core/recovery.py

```diff
@@ -70,6 +70,14 @@ def recover_action_run(action_run, action_runner):
     # and updates its internal state acoording to its result.
     action_run.watch(recovery_action_command)
 
+    if not action_run.machine.check('running'):
+        log.error(
```

Comment:
don't we want to error out of here? i assume if action run is not `running`, killing it won't kill the recovered process and other similar edge cases?

Reply:
I think it'll be ok? you can still move to a terminal state from unknown.